com.napster.napster-bigscreen-electron.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.napster.napster-bigscreen-electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+separate-locales: false
+command: start-napster-bigscreen-electron
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=/run/udev:ro
```

Comment:
why do you need this?

Reply:
This was required to make the app able to be controlled by the steamdeck/gamepad controller. That together with `--device=all`.

Reply:
Just to reference another web app requiring the `--filesystem=/run/udev:ro`:
- https://www.gamingonlinux.com/2022/03/microsoft-announce-xbox-cloud-gaming-for-steam-deck-with-edge/
- https://support.microsoft.com/en-gb/topic/xbox-cloud-gaming-in-microsoft-edge-with-steam-deck-43dd011b-0ce8-4810-8302-965be6d53296